roles/ara_api/tasks/main.yaml

```diff
@@ -22,6 +22,11 @@
     - "{{ ansible_facts['distribution'] }}.yaml"
     - "{{ ansible_facts['os_family'] }}.yaml"
 
+- name: Prime path_with_virtualenv when not using venvs
+  set_fact:
+    path_with_virtualenv: "{{ ansible_facts['env']['PATH'] }}"
+  when: not ara_api_venv | bool
```

Comment:
Is there a reason why you put this here instead of under ``install/pypi.yml`` and ``install/source.yml`` ?

Reply:
Yes, this variable is used in multiple locations and I ran into issues when installing it without venv's on RHEL7 and it broke on that var being undefined. :-)

Reply:
```
$ grep -rl path_with_virtualenv
roles/ara_api/tasks/config.yaml
roles/ara_api/tasks/database_engine/django.db.backends.sqlite3.yaml
roles/ara_api/tasks/database_engine/django.db.backends.postgresql.yaml
roles/ara_api/tasks/database_engine/django.db.backends.mysql.yaml
roles/ara_api/tasks/install/pypi.yaml
roles/ara_api/tasks/install/source.yaml
roles/ara_api/tasks/install/distribution.yaml
roles/ara_api/tasks/wsgi_server/gunicorn.yaml
roles/ara_api/tasks/main.yaml # << this one I added
```